attestation-agent/lib/src/lib.rs

```diff
@@ -178,21 +185,28 @@ impl AttestationAPIs for AttestationAgent {
 
     #[allow(unreachable_code)]
     async fn get_token(&mut self, _token_type: &str) -> Result<Vec<u8>> {
-        let _params = aa_kbc_params::get_params().await?;
+        let _url = match aa_kbc_params::get_params().await {
+            Ok(params) => params.uri().to_string(),
+            Err(_) => {
+                let config = config::Config::try_from(self.config_file_path.as_str())
```

Comment:
we probably don't want to re-read the configuration file on every get_token() call, maybe we can do this lazily or better yet, have the configuration option on the attestation-agent binary as a cli param and use it as initialization param for the  AA struct:

```rust
AttestationAgent {
    kbc_module_list: KbcModuleList::new(),
    kbc_instance_map: HashMap::new(),
    config: Option<config::Config>,
}
```